tools/vscode/.vscode/launch.json

```diff
@@ -16,6 +16,18 @@
 				"${workspaceFolder}/out/**/*.js"
 			],
 			"preLaunchTask": "${defaultBuildTask}"
-		}
+		},
+		{
+            "name": "Launch Extension",
+            "type": "extensionHost",
+            "request": "launch",
+            "args": [
+                "/Users/rjhala/research/vtock"
```

Comment:
?

Reply:
eeks, didn't realize that was in the git!